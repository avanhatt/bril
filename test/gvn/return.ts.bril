test/ts/return.ts.bril