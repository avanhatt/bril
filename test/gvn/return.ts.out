void main() {
b1:
  v01: int = const 1;
  x1: int = id v01;
  v11: int = const 2;
  y1: int = id v11;
  v6: int = call add2(x1, y1);
  print v6;
  ret ;
}
int add2(x : int, y : int) {
b1:
  ret v21;
}
